applications/nrf_desktop/src/modules/Kconfig.led_stream

```diff
@@ -15,7 +15,7 @@ config DESKTOP_LED_STREAM_ENABLE
 config DESKTOP_LED_STREAM_QUEUE_SIZE
 	int "Stream led event queue size"
 	depends on DESKTOP_LED_ENABLE && DESKTOP_CONFIG_CHANNEL_ENABLE
-	default 5
+	default 15
 	range 2 255
```

Comment:
Isn't the range now 1-254 ?

Reply:
Yes, you are right. I missed it. I will fix.